debug: true

# Global and Local Planner can be set in scatwheel_navigation/config/planners.yaml

mode: "shared"
# Options: 
# autonomous 
# shared 
# manual
# docked

base_global_planner: global_planner/GlobalPlanner
# Options: 
# global_planner/GlobalPlanner 
# navfn/NavfnROS
# carrot_planner/CarrotPlanner
# global_planner_plugin/GlobalPlannerPlugin 

base_local_planner: base_local_planner/TrajectoryPlannerROS
# Options: 
# dwa_local_planner/DWAPlannerROS 
# base_local_planner/TrajectoryPlannerROS
# teb_local_planner/TebLocalPlannerROS

switch_threshold: 0.6

sc_type: "dwa"
#Options:
# dwa, mpc, trajectory, goal
